setup/cluster/README.md

```diff
@@ -0,0 +1,228 @@
+
+### 4. Deploy Connaiseur
+
+Deploying Connaisseur is easy once the configuration is completed. Set your Kubernetes context to the correct cluster and run:
+
+```bash
+make install
+```
+
+Your output should look something like the following:
+
+```bash
+bash helm/certs/gen_certs.sh
+Generating RSA private key, 4096 bit long modulus (2 primes)
+...++++
+...............................................................................................................................................................................................................................................................................................................................................++++
+e is 65537 (0x010001)
+Signature ok
+subject=CN = connaisseur-svc.connaisseur.svc
+Getting Private key
+kubectl create ns connaisseur || true
+namespace/connaisseur created
+kubectl config set-context --current --namespace connaisseur
+Context "your-kubernetes-context" modified.
+helm install connaisseur helm --wait
+NAME: connaisseur
+LAST DEPLOYED: Wed Oct  7 10:45:53 2020
+NAMESPACE: connaisseur
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+Connaisseur was successfully deployed.
+
+### 5. (Optional) Verify Connaisseur is working
+
+If you were just trusting everything someone told you, you wouldn't be here looking for a tool that ensures image integrity, so don't take our word for Connaisseur working. Go ahead and try it:
+
+`kubectl get all -n connaisseur` will show you most* of what you deployed (ConfigMaps, Secrets and MutatingWebhookConfiguration are not shown).
+
+Try deploying the images you created in step 2. First try deploying the unsigned image:
+
+```bash
+kubectl create -f - << EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sample-deployment
+  labels:
+    app: sample
+spec:
+  selector:
+    matchLabels:
+      app: sample
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: sample
+    spec:
+      containers:
+      - name: sample
+        image: ${IMAGE_PATH}:unsigned
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 5000
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: sample
+  labels:
+    app: sample
+spec:
+  ports:
+  - name: port
+    port: 8080
+    targetPort: 5000
+  selector:
+    app: sample
+  type: LoadBalancer
+EOF
+```
+
+You should see the deployment being rejected with an error similar to
+
+```bash
+service/sample created
+Error from server: error when creating "STDIN": admission webhook "connaisseur-svc.connaisseur.svc" denied the request: could not find signed digest for image "docker.io/testingconny/testimage:unsigned" in trust data.
+```
+
+> Note that the service was indeed created, because it does not reference an image and is thus not blocked by Connaisseur. You can clean it up by calling `kubectl delete service sample`.
```

Comment:
```suggestion
> Note that while the container is blocked Kubernetes still creates the service, because it does not reference an image and is thus not denied by Connaisseur. You can clean it up by calling `kubectl delete service sample`.
```
otherwise it reads a bit like Conny is not working